aixplain/factories/agent_factory/__init__.py

```diff
@@ -136,7 +136,7 @@ def create(
             raise Exception("Agent Onboarding Error: Please contact the administrators.")
 
         if 200 <= r.status_code < 300:
-            agent = build_agent(payload=response, tools=tools, api_key=api_key)
+            agent = build_agent(payload=response, api_key=api_key)
```

Comment:
I don't think we should remove tools here because if we remove them, we'll load all the tools again, which will increase the agent creation time.

Reply:
Changed according to suggestion